backend/src/main/java/com/votogether/domain/member/dto/MemberNicknameRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.votogether.domain.member.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+
+@Schema(description = "닉네임 변경 요청")
+public record MemberNicknameRequest(
```

Comment:
`P3`
```suggestion
public record MemberNicknameUpdateRequest(
```
어떤 행위에 대한 요청값인지 명시해주는 것은 어떨까요?!

Reply:
수정하겠습니다!